migrating_from_ocp_3_to_4/index.adoc

```diff
@@ -0,0 +1,86 @@
+[id="migration-from-version-3-to-4-overview"]
+= Migration from version 3 to 4 overview
+include::modules/common-attributes.adoc[]
+:context: migration-from-version-3-to-4-overview
+
+toc::[]
+
+{product-title} 4 clusters are different from {product-title} 3 clusters. {product-title} 4 clusters contain new technologies and functionality that result in a cluster that is self-managing, flexible, and automated. To learn more about migrating from {product-title} 3 to 4 see xref:../migrating_from_ocp_3_to_4/about-migrating-from-3-to-4.adoc#about-migrating-from-3-to-4[about migrating from OpenShift Container Platform 3 to 4].
+
+[id="mtc-3-to-4-overview-differences-mtc"]
+=== Differences between {product-title} 3 and 4
+Before migrating from {product-title} 3 to 4, you can check xref:../migrating_from_ocp_3_to_4/planning-migration-3-4.adoc#planning-migration-3-4[differences between {product-title} 3 and 4]. Review the following information:
+
+** Architecture
+** Installation and upgrade
+** Storage, network, logging, security, and monitoring considerations
+
+[id="mtc-3-to-4-overview-planning-network-considerations-mtc"]
+=== Planning network considerations
+Before migrating from {product-title} 3 to 4, review the following information to xref:../migrating_from_ocp_3_to_4/planning-migration-3-4.adoc#migration-preparing-networking[networking considerations]:
+
+** DNS considerations
+*** Isolating the DNS domain of the target cluster from the clients.
+*** Setting up the target cluster to accept the source DNS domain.
+
+You can migrate stateful application workloads from {product-title} 3 to 4 at the granularity of a namespace. To learn more about MTC see xref:../migrating_from_ocp_3_to_4/about-mtc-3-4.adoc#about-mtc-3-4[understanding MTC].
+
+[NOTE]
+====
+If you are migrating from {product-title} 3, see xref:../migrating_from_ocp_3_to_4/about-migrating-from-3-to-4.adoc#about-migrating-from-3-to-4[About migrating from {product-title} 3 to 4] and xref:../migrating_from_ocp_3_to_4/installing-3-4.adoc#migration-installing-legacy-operator_installing-3-4[Installing the legacy {mtc-full} Operator on {product-title} 3].
+====
+
+[id="mtc-overview-install-mtc"]
+== Installing MTC
+Review the following tasks to install the MTC:
+
+. Install the {mtc-full} Operator on target cluster by using Operator Lifecycle Manager (OLM).
+* {product-title} 4.6 and later versions: xref:../migrating_from_ocp_3_to_4/installing-3-4.adoc#installing-3-4[Install the Migration Toolkit for Containers Operator by using OLM].
```

Comment:
Should "{product-title} 4.6 and later versions: " be removed completely, based on @apinnick's comment that we assume that they are on the current OCP version?

Reply:
Yes, @bergerhoffer is correct.

Reply:
acked